src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends Ball {
+    public Ball getRandomBall() {
+        Ball ball = new Ball();
+        ColorSupplier colorSupplier = new ColorSupplier();
+        String colorRandom = colorSupplier.getRandomColor();
+        int number = new Random().nextInt(100);
```

Comment:
Move "Random" to the class field.
Avoid to create new Random each time when getRandomBall() is invoked.